x/group/client/testutil/tx.go

```diff
@@ -2306,7 +2223,213 @@ func (s *IntegrationTestSuite) TestTxLeaveGroup() {
 	}
 }
 
-func (s *IntegrationTestSuite) getGroupIdFromTxResponse(txResp sdk.TxResponse) string {
+func (s *IntegrationTestSuite) TestExecProposalsWhenMemberLeavesOrIsUpdated() {
+	val := s.network.Validators[0]
+	clientCtx := val.ClientCtx
+
+	weights := []string{"1", "1", "2"}
+	accounts := s.createAccounts(3)
+	testCases := []struct {
+		name      string
+		votes     []string
+		members   []string
+		malleate  func(groupID string) error
+		expectErr bool
+		errMsg    string
```

Comment:
this naming is a bit misleading, we're not strictly expecting an error but rather some logs about the proposal execution failure

Reply:
True